Shared/Coordinators/VideoPlayerCoordinator.swift

```diff
@@ -52,9 +52,7 @@ final class VideoPlayerCoordinator: NavigationCoordinatable {
 
         #else
         if Defaults[.VideoPlayer.videoPlayerType] == .swiftfin {
-            PreferencesView {
```

Comment:
Thanks for taking a look at this, however I'll take a while because I don't have my Apple TV with me personally and I'll have to think/see if this is the route to go because SwiftUI `onExitCommand/on_Command` has specific rules for how they get called. For example for what you've done in this PR it actually crashes because the view, without the overlay, is not in focus so SwiftUI will just exit the player without calling our `onExitCommand`. Focus on tvOS+SwiftUI is a mess and atow, none of what you've done works in the simulator for me because the view isn't in "focus", so SwiftUI handles all of the commands itself.

The purpose of `PreferencesView` is to intercept remote presses and not let SwiftUI handle them (but this isn't implemented yet). I would have also looked at implemented detecting swipes from the 1st gen Siri Remote at that layer.

Here's a super old commit when the video player was in UIKit where we handled the key presses:

- https://github.com/jellyfin/Swiftfin/blob/5d0f933a2c083fd27ae4579960042cc51cf22450/Swiftfin%20tvOS/Views/VideoPlayer/VLCPlayerViewController.swift#L206